Source/WebCore/animation/AnimationTimelinesController.cpp

```diff
@@ -310,42 +311,46 @@ void AnimationTimelinesController::maybeClearCachedCurrentTime()
         m_cachedCurrentTime = std::nullopt;
 }
 
-void AnimationTimelinesController::registerNamedScrollTimeline(const AtomString& name, Element& source, ScrollAxis axis)
+static Element* originatingElement(Ref<ScrollTimeline> timeline)
```

Comment:
This could be a `Ref<ScrollTimeline>&`. But really I think we can do away with this static method entirely, it only really needs to be used once in `unregisterNamedTimeline()` since the other call sites are places where we're already dealing with logic specific to `ScrollTimeline` or `ViewTimeline`.

Reply:
Removed.